pages/updatePaymentMethod.js

```diff
@@ -0,0 +1,323 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import gql from 'graphql-tag';
+import styled from 'styled-components';
+import { graphql } from 'react-apollo';
+import { fontSize, maxWidth } from 'styled-system';
+import { Flex, Box } from '@rebass/grid';
+import { FormattedMessage, injectIntl } from 'react-intl';
+import { get, cloneDeep } from 'lodash';
+
+import { withUser } from '../components/UserProvider';
+import Header from '../components/Header';
+import Body from '../components/Body';
+import Footer from '../components/Footer';
+import Container from '../components/Container';
+import { H1, H5 } from '../components/Text';
+import NewCreditCardForm from '../components/NewCreditCardForm';
+import Loading from '../components/Loading';
+
+import { getStripe, stripeTokenToPaymentMethod } from '../lib/stripe';
+
+import { compose } from '../lib/utils';
+import { withStripeLoader } from '../components/StripeProvider';
+
+import { getSubscriptionsQuery } from '../lib/graphql/queries';
+
+import StyledButton from '../components/StyledButton';
+import RedeemBackground from '../components/virtual-cards/RedeemBackground';
+
+const ShadowBox = styled(Box)`
+  box-shadow: 0px 8px 16px rgba(20, 20, 20, 0.12);
+`;
+
+const Subtitle = styled(H5)`
+  color: white;
+  text-align: center;
+  margin: 0 auto;
+  ${fontSize};
```

Comment:
This prop is already supported natively by `H5`

Reply:
What do you mean? To not use the fontSize and maxWidth from the styled-system? I have tried to add these props to the Subtitle directly and they don't seem to be applying 🤔

Reply:
I think @Betree is just saying that ` ${fontSize};` can be removed because it's already supported in `H5`.